rose/web/game.js

```diff
@@ -400,6 +402,28 @@ var ROSE = (function() {
         }
     }
 
+    function SoundController(loader) {
+        this.loader = loader;
+        this.sounds = [null];
+        var self = this;
+
+        loader.load("res/soundtrack/Nyan_Cat.ogg", function (sound) {
+            sound.loop = true;
+            self.sounds[0] = sound;
+            self.toggle_sound(0);
+        });
+    }
+
+    SoundController.prototype.toggle_sound = function(sound_id) {
```

Comment:
I'm not sure how you are going to use this in the next version, since when clicking sound button, we have to:
- play/pause sound
- replace the icon

It seems that the entire sound controller belongs to the next PR, this PR is happy with the plain loader, playing the file when it was loaded, as in the first commit.